webapp/src/components/viewHeader/newCardButtonTemplateItem.tsx

```diff
@@ -12,30 +12,44 @@ import EditIcon from '../../widgets/icons/edit'
 import OptionsIcon from '../../widgets/icons/options'
 import Menu from '../../widgets/menu'
 import MenuWrapper from '../../widgets/menuWrapper'
+import CheckIcon from '../../widgets/icons/check'
+import {BoardView} from '../../blocks/boardView'
 
 type Props = {
+    boardView: BoardView
     cardTemplate: Card
     addCardFromTemplate: (cardTemplateId: string) => void
     editCardTemplate: (cardTemplateId: string) => void
 }
 
 const NewCardButtonTemplateItem = React.memo((props: Props) => {
-    const {cardTemplate} = props
+    const {boardView, cardTemplate} = props
     const intl = useIntl()
     const displayName = cardTemplate.title || intl.formatMessage({id: 'ViewHeader.untitled', defaultMessage: 'Untitled'})
+
+    const icon = boardView.defaultTemplateId === cardTemplate.id ? <CheckIcon/> : cardTemplate.icon
+
     return (
         <Menu.Text
             key={cardTemplate.id}
             id={cardTemplate.id}
             name={displayName}
-            icon={<div className='Icon'>{cardTemplate.icon}</div>}
+            icon={icon}
             onClick={() => {
                 props.addCardFromTemplate(cardTemplate.id)
             }}
             rightIcon={
                 <MenuWrapper stopPropagationOnToggle={true}>
                     <IconButton icon={<OptionsIcon/>}/>
                     <Menu position='left'>
+                        <Menu.Text
+                            icon={<CheckIcon/>}
+                            id='default'
+                            name={intl.formatMessage({id: 'ViewHeader.set-default-template', defaultMessage: 'Set as default'})}
+                            onClick={async () => {
+                                await mutator.setDefaultTemplate(boardView, cardTemplate.id)
```

Comment:
We need a way to clear the current default template:
* Add a "set as default" sub-menu option to the "Empty Card" option, which should clear the default template ID.

Reply:
@chenilim I really like this idea 👍

Reply:
I'll try this.
And another thing of "Empty Card" icon, I think the icon should be replaced. The current icon always cause me think it is not loaded, and it's a broken icon... Looks very strange...

![image](https://user-images.githubusercontent.com/13222125/127598759-b3e478f8-202e-44fd-a2f7-11223ba55c40.png)